src/Symfony/Component/Finder/Iterator/RecursiveDirectoryIterator.php

```diff
@@ -118,8 +118,10 @@ public function rewind()
             return;
         }
 
-        // @see https://bugs.php.net/bug.php?id=49104
-        parent::next();
+        // @see https://bugs.php.net/bug.php?id=68557
+        if (PHP_VERSION_ID < 50523 || PHP_VERSION_ID >= 50600 && PHP_VERSION_ID < 50607) {
```

Comment:
@dunglas, Do you think this one also needs to be changed to a guard clause? I think that the same condition should be used in both cases, but here duplication of the parent::rewind call in the guard clause and in the normal flow would look weird.